src/modules/personenkontext/api/personenkontext.controller.ts

```diff
@@ -75,7 +80,21 @@ export class PersonenkontextController {
     @ApiInternalServerErrorResponse({ description: 'An internal server error occurred.' })
     public async findPersonenkontextById(
         @Param() params: FindPersonenkontextByIdParams,
+        @Permissions() permissions: PersonPermissions,
     ): Promise<PersonendatensatzResponseAutomapper> {
+        {
+            // Check permissions
+            const result: Result<unknown, DomainError> = await this.personenkontextRepo.findByIDAuthorized(
```

Comment:
Please throw the corresponding error MissingPermissionsError and do not handle it via SchulConnexErrorMapper.

Annotating the possible errors in a way like in OrganisationController helps the FE to identify possible errors:

 e.g. @ApiBadRequestResponse({ description: 'Request has wrong format.', type: DbiamOrganisationError })